### ISSUE TITLE
Compilation Error when try to run yarn start

### ISSUE THREAD

**Issue by kousherAlam** [InseeFrLab/keycloakify#68]
When i run the `yarn start` then i got the following error 

```
Compiled with problems:X

ERROR in ./node_modules/keycloakify/lib/components/LoginOtp.js 43:13-28

Module not found: Error: Can't resolve 'path' in '/home/kousher/Projects/edge-commerce/keycloak-bookstore/node_modules/keycloakify/lib/components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
	- install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "path": false }


ERROR in ./node_modules/keycloakify/lib/components/Template.js 75:13-28

Module not found: Error: Can't resolve 'path' in '/home/kousher/Projects/edge-commerce/keycloak-bookstore/node_modules/keycloakify/lib/components'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
	- install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "path": false }


ERROR in ./node_modules/keycloakify/lib/getKcContext/kcContextMocks/kcContextMocks.js 70:13-28

Module not found: Error: Can't resolve 'path' in '/home/kousher/Projects/edge-commerce/keycloak-bookstore/node_modules/keycloakify/lib/getKcContext/kcContextMocks'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
	- install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "path": false }


ERROR in ./node_modules/keycloakify/lib/getKcContext/kcContextMocks/urlResourcesPath.js 8:13-28

Module not found: Error: Can't resolve 'path' in '/home/kousher/Projects/edge-commerce/keycloak-bookstore/node_modules/keycloakify/lib/getKcContext/kcContextMocks'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
	- install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "path": false }
```

OS: Ubuntu 20.04.3 LTS 64 Bit

**Comment by kousherAlam** [comment]
I tried with `yarn add path-browserify`. But still got the same issue.

**Comment by garronej** [comment]
Hi @kousherAlam,  
I think the problem is specific yo your setup. `path-browserify` is a dependency of `keycloakify` I don't see why it wouldn't resolve.  
Have you tried deleting the `node_modules` directory and the `.yarn.lock` and reinstalling from scratch?

**Comment by kousherAlam** [comment]
![image](https://user-images.githubusercontent.com/14360099/146419493-fce0602a-4be1-41a3-a707-8ccbe1d86ce8.png)

hi @garronej you can see in the screenshot everything seems okay. but the i'm still getting the issue,

**Comment by garronej** [comment]
ok hold on I will see if I can reproduce updating to the latest version of Webpack.

**Comment by garronej** [comment]
That's a [create-react-app issue](https://github.com/facebook/create-react-app/issues/11756)

**Comment by garronej** [comment]
Considering [this statement](https://github.com/facebook/create-react-app/issues/11756#issuecomment-994680220) (I disagree btw) I will avoid using `path` in the frontend part.  
Hold on...

**Comment by garronej** [comment]
The bug with path is gone but there are other errors with Webpack 5. Fixing...

**Comment by garronej** [comment]
It works now but there  is still a bug with fonts... fixing

**Comment by garronej** [comment]
Fixed in 4.2.18
